doc/manual/variables-and-scoping.rst

```diff
@@ -386,24 +445,29 @@ existing value location, and ``let`` creates new locations. This
 difference is usually not important, and is only detectable in the
 case of variables that outlive their scope via closures. The ``let``
 syntax accepts a comma-separated series of assignments and variable
-names::
+names:
 
-    let var1 = value1, var2, var3 = value3
-        code
-    end
+.. doctest::
+
+    julia> let x = 1, y = 3, z = 4
```

Comment:
It is also possible to not assign a value to a variable, as in the original example.  Maybe
```
julia> x, y, z = -1, -1, -1;

julia> let x =1, z
           println("x: $x, y: $y") # x is local variable, y the global
           println("z: $z") # errors as z has not been assigned yet but is local
       end
x: 1, y: -1
ERROR: UndefVarError: z not defined
```